.dictionary.txt

```diff
@@ -1,2 +1,3 @@
+Gitleaks
 hhatto
```

Comment:
Commit message: recommend clarifying that the additional arguments passed via `--log-opts` are ferried to `git log`. In other words, let's not assume familiarity with the purpose of the `--log-opts` argument.